howso/scikit/scikit.py

```diff
@@ -751,6 +751,21 @@ def describe_prediction(self, X, details=None) -> Dict:
                 are outside the min or max of the corresponding feature values
                 of all the cases in the local model area. Uses only the context
                 features of the reacted case to determine that area.
+            - prediction_stats : bool, optional
+                When true outputs feature prediction stats for all (context
+                and action) features locally around the prediction. The stats
+                returned  are ("r2", "rmse", "spearman_coeff", "precision",
+                "recall", "accuracy", "mcc"). Confusion matrices may also be
+                returned by setting 'confusion_matrix' to true. Uses only the
```

Comment:
'confusion_matrices' here, not 'confusion_matrix'. Maybe should also be double quotes? idk

Reply:
applies to the other spots too